index.html

```diff
@@ -1,19 +1,94 @@
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header class="header">
+      <img
+        src="/img/karma-logo.svg"
+        id="logo-image-id"
+        class="logo-image"
+        alt="image_of_logo"
+      />
+      <nav class="navigation">
+        <a href="#" class="nav-links">Meet Karma</a>&nbsp;
+        <a href="#" class="nav-links">How it Works</a>&nbsp;
+        <a href="/store.html" class="nav-links">Store</a>&nbsp;
+        <a href="#" class="nav-links">Blog</a>&nbsp;
+        <a href="#" class="nav-links">Help</a>&nbsp;
+        <a href="#" class="nav-links">Logic</a>
+      </nav>
+    </header>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <main class="main-section">
+      <section class="backgroundImage-and-title">
+        <div class="intro">
+        <h1 class="title">Introducing Karma</h1>
+        <p class="intro-paragraph">Bring WiFi with you, everywhere you go</p>
+        <button class="learn-more-btn btn">Learn more</button>
+        </div>
+      </section>
 
-</body>
+      <section class="product-section">
+        <h2>Everyone needs a little Karma</h2>
+        <div class="sub-product-container">
+          <div class="individual-product-container">
+                <img src="/img/icon-devices.svg" class="product-image" alt="image_of_devices"/>
+            <p>Internet for all devices</p>
+          </div>
+
+          <div class="individual-product-container">
+                <img src="/img/icon-coffee.svg" class="product-image" alt="image_of_coffee_cup"/>
+            <p>Boost your productivity</p>
+          </div>
+
+          <div class="individual-product-container">
+                <img src="/img/icon-refill.svg" class="product-image" alt="image_of_fuel"/>
+            <p>Pay as You Go</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="connection_coverage">
+        <img class="image_for_homepage_feature" src="/level-2/homepage-feature.png">
+      <div class="go_karma_today_section">
+        <p>"Where I am, I just don't worry about my connection anymore"</p>
+      <button class="Go_karma_btn btn"><a class="Go_karma_btn" href="/store.html">Get Karma today</a></button>
+      </div>
+      </section>
+
+    </main>
+
+    <footer class="footer">
```

Comment:
the social media icons need to be smaller and they need to have a circular border around them.

Reply:
Sure. thanks